contracts/conduit/ConduitController.sol

```diff
@@ -154,8 +154,11 @@ contract ConduitController is ConduitControllerInterface {
         } else if (!isOpen && channelPreviouslyOpen) {
             // Set a previously open channel as closed via "swap & pop" method.
             // Decrement located index to get the index of the closed channel.
-            uint256 removedChannelIndex = channelIndexPlusOne - 1;
-
+            uint256 removedChannelIndex;
+            // channelPreviouslyOpen already ensures channelIndexPlusOne > 0
+            unchecked {
+                removedChannelIndex = channelIndexPlusOne - 1;
```

Comment:
This looks safe

Reply:
To clarify: this is safe.
In this branch, `channelPreviouslyOpen` must be true, which means that `channelIndexPlusOne != 0`. Since it has type `uint`, it must be `> 0` therefore safe.